tests/feature/controllers/group.test.js

```diff
@@ -0,0 +1,265 @@
+import Group from '../../../src/db/models/group';
+import { GroupNotFoundError } from '../../../src/exceptions/errors';
+import * as AuthService from '../../../src/services/auth';
+import * as GroupService from '../../../src/services/group';
+
+
+const supertest = require('supertest');
+
+const { app } = require('../../../src/app');
+
+// Create our Test AuthToken
+const authToken = AuthService.createToken({
+    id: '6f7a7188-5b7d-4dbb-8a43-12031b94971e',
+    login: 'John Doe',
+    password: 'secret',
+    age: 21,
+    isDeleted: false
+});
+
+describe('Test the groups controller', () => {
+    test('It should return all groups', async () => {
+        // Mock Model findAll
+        jest.spyOn(Group, 'findAll').mockImplementation(() => {
+            return Promise.resolve([
+                {
```

Comment:
I can see that these objects are the same across several tests.
Consider always preparing test dummies before at the beginning of the file and reusing them.